app/controllers/instance_user_role_requests_controller.rb

```diff
@@ -14,6 +20,21 @@ def create
     end
   end
 
+  def approve
+    @user_role_request.assign_attributes(user_role_request_params)
+
+    @success, instance_user = @user_role_request.approve_and_destroy!
+    if @success
+      flash.now[:success] = t('.success', user: instance_user.user.name, role: instance_user.role)
+    else
+      flash.now[:danger] = instance_user.errors.full_messages.to_sentence
```

Comment:
Actually, the only possible error here is "Role cannot be blank", the message is for that error. It will show when the user didn't select any role.  

It's not a random error that we don't know, and if in some extreme scenario that other errors occur, they cannot see this page also.